image_client/PIC_undo_batch.py

```diff
@@ -106,26 +137,24 @@ def casbotany_csv_purger(self, database: str, table: str, MD5: str):
 
         table_list = ['collectionobjectattachment', 'attachment',
                       'determination', 'collectionobject', 'collector',
-                      'collectingevent', 'locality', 'agent', ]
+                      'collectingevent', 'locality', 'agent']
 
         for table in table_list:
-            self.sql_time_purger(database='casbotany', table=table,
-                                 timestamp1=time_stamp_list[0],
-                                 timestamp2=time_stamp_list[1])
+            self.batch_undo_timestamps(database='casbotany', table=table,
```

Comment:
hardcoded database names - the way to do this, ideally, is that the connection itself specifies the database name. That way you can use a database-local user, and not an admin account to update these records. so you'd crfeate two db connections - one per database - with the relevant accounts. you pass the connection variable around, not the database name.